pkg/ingress/kube/ingressv1/controller.go

```diff
@@ -539,9 +542,22 @@ func (c *controller) ConvertHTTPRoute(convertOptions *common.ConvertOptions, wra
 							MatchType: &networking.StringMatch_Prefix{Prefix: path},
 						}
 					} else {
-						path = strings.TrimSuffix(path, "/")
+						// the path maybe prefix /xxx or prefix /xxx/
+						// we both convert it to prefix /xxx/ and exact /xxx
+						var extraExactPath string
+						if path[len(path)-1] != '/' { // prefix /xxx
```

Comment:
Why not trim the possible suffix "/" first, so the logic will look clearer

Reply:
done